akka-testkit/src/main/scala/akka/testkit/TestKit.scala

```diff
@@ -1015,19 +1015,25 @@ object TestKit {
    * Shut down an actor system and wait for termination.
    * On failure debug output will be logged about the remaining actors in the system.
    *
+   * The `duration` is dilated by the timefactor.
+   *
    * If verifySystemShutdown is true, then an exception will be thrown on failure.
    */
   def shutdownActorSystem(
```

Comment:
when this method was used directly the duration wasn't dilated and used default 10 seconds